makefiles/suit.v4.inc.mk

```diff
@@ -62,6 +71,9 @@ suit/notify: | $(filter suit/publish, $(MAKECMDGOALS))
 		--payload "$(SUIT_COAP_ROOT)/$$(basename $(SUIT_MANIFEST_SIGNED_LATEST))" && \
 		echo "Triggered $(SUIT_CLIENT) to update."
 
+suit/delkeys:
+	rm -f *key*
```

Comment:
this is unrelated and also strikes me as a bit extreme. ;)

Reply:
Yes should be in different PR, sorry. But would removing the specific file names be better then?

Reply:
:+1:

imagine the surprise when someone starts working on e.g., "hash_key.c" and runs "make delkeys" to get to a clean state before committing...